resources/views/website/layouts/assets/stylesheets.blade.php

```diff
@@ -1,5 +1,10 @@
-@if(Auth::check() && Auth::user()->theme)
+@if(Carbon::now()->month===Carbon::DECEMBER ||(config('app.env') != 'production'))
```

Comment:
First off, this won't be activated on production. But please also add the option for users to turn this off.

Reply:
We discussed this with Max and Jonathan but we did not want to make it a permanent theme, and we did not want to alter the database to make the choice of which theme you have (since if we remove it we will have to do a migration that everyone with theme 5 will be set back to theme 1 or another one), so we discussed it would be best to just force everyone over it for 1 month. Or do you have an easier way to do this?

Reply:
I personally did not mind adding another theme, and just running the following query on the database when the holidays are over:
```UPDATE saproto.users u
SET u.theme=1
WHERE u.theme = 4```
Or whatever the id of the theme will be.
I still believe this is the best solution.

Reply:
Added the ability to disable the theme by setting a cookie (button is found by the theme dashboard)